web-components/src/components/tabs/Tabs.ts

```diff
@@ -35,15 +38,15 @@ export namespace Tabs {
   type TabsViewportDataList = TabViewportData[];
 
   type TabId = Element["id"];
-
   @customElementWithCheck("md-tabs")
-  export class ELEMENT extends ResizeMixin(RovingTabIndexMixin(LitElement)) {
+  export class ELEMENT extends ResizeMixin(RovingTabIndexMixin(SlottedMixin(LitElement))) {
     @property({ type: Boolean }) justified = false;
     @property({ type: String }) overlowLabel = "More";
-    @property({ type: String }) direction = "horizontal";
+    @property({ type: Boolean }) draggable = false;
```

Comment:
Please organize all of these different properties, internalProperties and queries, it's getting hard to read.

Reply:
Definitely I will re-arrange and make it more user readable